.github/container/install-lingvo.sh

```diff
@@ -30,12 +16,16 @@ pushd ${LINGVO_INSTALLED_DIR}
 git fetch origin pull/329/head:pr329
 git config user.name "JAX Toolbox"
 git config user.email "jax@nvidia.com"
-# git cherry-pick pr326 pr328 pr329  ## pr326, pr328 merged
+# git cherry-pick --allow-empty pr326 pr328 pr329  ## pr326 pr328 merged
 git cherry-pick --allow-empty pr329
 
 # Disable 2 flaky tests here
 patch -p1 < /opt/lingvo.patch
+popd
 
+
+## Install lingvo
+pushd ${LINGVO_INSTALLED_DIR}
```

Comment:
Why popd followed by the opposite pushd?

Reply:
They where present for historical reasons. I'm removing them.